portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,28 +20,25 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private ArrayList<String> createList() {
-      ArrayList<String> greetings = new ArrayList<>();
-      greetings.add("Hello!");
-      greetings.add("How are you?");
-      greetings.add("Goodbye!");
-      return greetings;
```

Comment:
go/prefer-immutable-collections
This one can be a immutableList with one line construction. For example, https://www.geeksforgeeks.org/immutable-list-in-java/

Reply:
Unfortunately GitHub only shows the line you're referencing, if you're referring to the createList() function, that was part of the last step and is deleted in this one (not sure why they have us writing code just to delete it after), if so I will keep that in mind for next time!

Reply:
SG